### DISCUSSION TITLE
can't verify academic affiliation

### DISCUSSION THREAD

**Discussion by ynoplanetashka2** [community/community#140816] [GitHub Education] [answered]
### Hi everyone,

I tried to verify my academic affiliation multiple times, but always getting the same response

> The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote. If you believe that the school you have applied for has incorrect information that affects your current application e.g. incorrect email domain, please select ‘My selected school has incorrect or incomplete information e.g. domains or campus location’ option when submitting a GitHub Education support ticket.
> 
> Please ensure that your academic affiliation document contains your last name exactly as it appears in your [GitHub billing information](https://github.com/settings/billing/payment_information). If you have only a single legal name, enter it in both the first and last name fields of your GitHub billing information.
> 
> You appear not to be near any campus location for the school you have selected. Your academic affiliation documentation must indicate virtual learning if you are remote. You must configure your browser and operating system to allow location access. You may not use a VPN. If you believe that our information about your school is incomplete or incorrect e.g. missing campus location(s), then please select ‘My selected school has incorrect or incomplete information e.g. domains or campus location’ option when submitting a GitHub Education support ticket.
> 
> Please ensure that your academic affiliation document contains your first name exactly as it appears in your [GitHub billing information](https://github.com/settings/billing/payment_information). Variations of your name and nicknames are unacceptable.
> 
> You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name exactly as it appears in your academic affiliation document. Please do not use a variation of your name or a nickname. Once you have updated your profile information log out and log back into GitHub before re-applying.

I'm in dormitory holded by the university in which I'm studing, I don't use vpn, I completed github account info with my full name, my documents have my name exactly as in my github account info. I'm getting answer immediatly after appying to verify my academic status, so I'm sure no one is actually checking my verification application. Please, explain what I'm doing wrong, and what I have to do in order to verify my affiliation. 

I searched for similar questions, but in similar question problem were with university campus location info, so I hope wrong location info of my university will be fixed.

**Comment by davevad93** [comment, marked as answer]
Hi @ynoplanetashka2 ,

Given that you are submitting the application from your university, it's possible that GitHub may have incorrect information about your school's location.

Try contacting support by opening a ticket [here](https://support.github.com/contact/education), select "`My selected school has incorrect or incomplete information e.g. domains or campus location`" and explain your issue.

**Comment by theodoredespins** [comment]
It sounds frustrating! Maybe contacting your university's admin to confirm their listed details could help kind of like how [top legal advisers for UK's media industry](https://pearllemonlegal.com/top-media-law-firms-london-uk/) ensure accurate records in complex cases.
